testsuite/tests/weak-ephe-final/weaklifetime_par.ml

```diff
@@ -3,9 +3,6 @@
 
 let size = 1000;;
 let num_domains = 4;;
-let random_state = Domain.DLS.new_key Random.State.make_self_init
-
-let random_int = Random.State.int (Domain.DLS.get random_state)
```

Comment:
This test does use several domains, and I'm not sure that removing the random-state initialization completely is a good idea -- at least it is a change in behavior. If one wanted to match the previous behavior, they would need to run `Random.State.make_self_init` for each domain, probably in the driver loop at the end of the test.

Then one could argue that tests should rather be deterministic, and that starting from a fixed seed is better. Still I would expect this fixed seed to be different for each domain; with the current code, if I understand think correctly all domains are going to draw the same random sequence, and this feels fairly wrong.